pyspecProcScripts/load_data.py

```diff
@@ -690,10 +707,6 @@ def proc_spincore_generalproc_v1(
     #     should be nothing outside that
     if "ph_overall" in s.dimlabels:
         s.reorder("ph_overall")
-        s["ph_overall"] = -s["ph_overall"] / 4 + 1
-        s["ph_overall"] %= 1
-        s["ph_overall"] *= 4
-        s.sort("ph_overall")
```

Comment:
this conditional has nothing to do with the axis coordinates, and what you're doing here would be handled above.  (here's we're just saying that if there is a dimension called "overall', we want it on the outside)